source/basic.tex

```diff
@@ -1658,7 +1658,7 @@
 used in further processing.
 
 \pnum
-A program point $P$ is said to follow
+A program point $P$ is said to \grammarterm{follow}
```

Comment:
Looks wrong to me. The "follow" is not a grammar term. Perhaps we should index it as `\defnx{follow}{follow!program point}`.